identity/src/lib.rs

```diff
@@ -144,3 +144,12 @@ pub mod iota {
   #[doc(inline)]
   pub use identity_iota::try_did;
 }
+
+pub mod prelude {
+  //! Prelude of commonly used types for the identity crate
```

Comment:
```suggestion
  //! Prelude of commonly used types
```

since at this point you know you're dealing with the identity crate